doc/source/rllib/rllib-algorithms.rst

```diff
@@ -172,13 +172,13 @@ Importance Weighted Actor-Learner Architecture (IMPALA)
     :width: 750
 
     **IMPALA architecture:** In a training iteration, IMPALA requests samples from all EnvRunners asynchronously and the collected episode
-    samples are returned to the main algo process as ray references (rather than actual objects available on the local algo process).
-    These episode references are then passed to the Learner(s) for asynchronous updates of the model(s).
+    samples are returned to the main algorithm process as ray references (rather than actual objects available on the local process).
```

Comment:
If you could convert these to active voice to clarify who is doing the returning or passing. Is it RLlib?